devdoc/thandle_requirements.md

```diff
@@ -0,0 +1,135 @@
+# thandle_requirements
+================
+
+## Overview
+
+`thandle` is a module that provides a list of macros that help with ref counted handles.
+
+Given a type `LL` previously introduced by `MU_DEFINE_STRUCT`, `thandle`'s macros will encapsulate the type, provide a HANDLE (that is, opaque type) to it which other APIs can use. The handle type is `THANDLE(LL)`.
```

Comment:
>LL [](start = 14, length = 2)

T

Reply:
changed

Reply:
old remains are old. changed.

---
In reply to: [385363919](https://github.com/Azure/azure-c-util/pull/16#discussion_r385363919) [](ancestors = 385363919)